src/Config/ConfigLoader.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Robo\Config;
+
+use Grasmash\YamlExpander\Expander;
+
+/**
+ * Load configuration files, and fill in any property values that
+ * need to be expanded.
+ */
+class ConfigLoader implements ConfigLoaderInterface
+{
+    protected $config = [];
+    protected $source = '';
+
+    public function getSourceName()
+    {
+        return $this->source;
+    }
+
+    public function setSourceName($source)
+    {
+        $this->source = $source;
+        return $this;
+    }
+
+    public function import($data)
+    {
+        $this->config = [];
```

Comment:
This prevents an implementer from calling YamlConfigLoader->load() multiple times, because it wipes out the previous configuration for each newly imported array.